configs/default/TTRH-TRANSTECF411_RPMFILTER

```diff
@@ -0,0 +1,86 @@
+# Betaflight / STM32F411 (S411) 4.1.0 Sep  5 2019 / 19:50:56 (2553a4a1f) MSP API: 1.42
+
+board_name TRANSTECF411
```

Comment:
I don't think that commenting out the `LED_STRIP` resource is a valid reason to create a different target variant (TTRH-TRANSTECF411_RPMFILTER). If we were going down that path we'd end up with all kinds of target variants for alternate functions on flight controllers. Alternatives like one that enables softserial on a LED pin; one that resources camera control to motor 6; and so on. This squarely falls into the manufacturer documentation responsibility.

Reply:
Ok - I've removed it. It just comes up a lot recently. I think I've seen like 5 people with that issue last week. Fortunately there are only few boards which we can't fix in a non conflicting way.